src/sentry/seer/anomaly_detection/utils.py

```diff
@@ -17,11 +18,17 @@
 from sentry.snuba.models import SnubaQuery, SnubaQueryEventType
 from sentry.snuba.referrer import Referrer
 from sentry.snuba.sessions_v2 import QueryDefinition
-from sentry.snuba.utils import get_dataset
+from sentry.snuba.utils import DATASET_OPTIONS, get_dataset
 from sentry.utils.snuba import SnubaTSResult
 
 NUM_DAYS = 28
 
+SNUBA_QUERY_EVENT_TYPE_TO_STRING = {
```

Comment:
Should this be a StrEnum instead of a dictionary?

Reply:
I don't think it can be, `SnubaQueryEventType.EventType` is already an enum: https://github.com/getsentry/sentry/blob/master/src/sentry/snuba/models.py#L79-L82

I tried but it errored `AttributeError: cannot reassign member 'ERROR'` - I guess you can't enum an enum

Reply:
🤔 you should be able to - i think the because the other is an `Enum` rather than a `StrEnum` it requires `ERROR.value` to access the string, rather than the enum value instance.

Reply:
I tried that too :/

Reply:
😭 - okay, thanks for trying!